common/scala/src/main/scala/whisk/core/database/ArtifactStore.scala

```diff
@@ -28,6 +28,14 @@ import whisk.common.Logging
 import whisk.common.TransactionId
 import whisk.core.entity.DocInfo
 
+abstract class StaleParameter(val value: Option[String])
+
```

Comment:
Ugh, right. Should've looked up the whole pattern. Only relevant if you want to match against them and have the compiler check for an incomplete match.